### DISCUSSION TITLE
How to get the repo's pull request with their linked issues?

### DISCUSSION THREAD

**Discussion by Garito** [octokit/octokit.js#1887] [Q&A]
Is it possible?

I can't even find the field that would related both
I list the issues and the pull request but can't see how they relate

Thanks

**Comment by gr2m** [comment]
With "liked issues" you mean issues that have been closed automatically when pull requests were closed?

We do something similar in `@semantic-release/github`, the relevant code is here:
https://github.com/semantic-release/github/blob/24344df17ad715f1d7fafcd3ced1283b9e1f4d8f/lib/success.js#L64-L76. We use my fellow @semantic-release co-maintainer Pierre's `issue-parser` package to parse a text body and find things like closing keywords: https://github.com/pvdlg/issue-parser

It's been a while since we created the code though, maybe there are better ways, make sure to check out the GraphQL API. A good place to ask "How do I XYZ with GItHub's APIs" is github.community/

If you find out something new, please share it with us :)

**Comment by Garito** [comment]
My research has been very disappointing so far
Apparently the api doesn't provide the information or a way to notice the link

I have not searched with graphql but I will check out your suggestions, thanks

Perhaps someone has more info than me
